docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,41 @@
 title: Changelog
 ---
 
+## 12.14.0
+
+_Released 06/07/2023_
+
+**Features:**
+
+- A new testing type switcher has been added to the Spec Explorer to make it
+  easier to move between E2E and Component Testing. An informational overview of
+  each type is displayed if it isn't configured in your project to help
+  newcomers to each testing type. Addresses
+  [#26448](https://github.com/cypress-io/cypress/issues/26448),
+  [#26836](https://github.com/cypress-io/cypress/issues/26836) and
+  [#26837](https://github.com/cypress-io/cypress/issues/26837).
```

Comment:
Generally I think we should prob have a single issue in the future for feature work that is is broken into multiple issues / PR that summarizes the primary goal.

Reply:
Originally this only referenced one issue, but the linter complained because the PR mentioned closing multiple.

Reply:
Was there a single github issue the feature branch closed though?

Reply:
Didn't appear that way when looking at the feature branch. The feature branch seemed to be acting as the root issue.

Reply:
Yeah for long running features a single issue would be the norm, like v13 release, it would be the root reference.Then we can track on the board with a single issue 🤷🏻‍♀️